src/main/java/org/broadinstitute/hellbender/tools/walkers/ReferenceConfidenceVariantContextMerger.java

```diff
@@ -194,18 +201,18 @@ private VCWithNewAlleles(final VariantContext vc, final List<Allele> newAlleles,
             this.isSpanningEvent = isSpanningEvent;
         }
 
-        public List<Allele> filterAllelesForFinalSet() {
+        public Stream<Allele> filterAllelesForFinalSet() {
```

Comment:
Ah, I didn't notice it was in a private inner class -- in that case, this is fine.